src/test/java/formflow/library/utilities/AbstractBasePageTest.java

```diff
@@ -46,6 +46,8 @@ public abstract class AbstractBasePageTest {
 
   @BeforeEach
   protected void setUp() throws IOException {
+    driver.manage().deleteAllCookies();
+    driver.resetInputState();
     initTestPage();
```

Comment:
I'm not 100% sure what the two lines above this are doing, but I wanted to clear out session information when `setup()` was called and this seems to do that.